frontend/src/components/document/DocumentListItem.tsx

```diff
@@ -19,9 +21,12 @@ type TProps = {
 };
 
 export const DocumentListItem: FC<TProps> = ({ children, listItem }) => {
-  const { slug, country_code, country_name, description, name, document_year, category } = listItem;
+  const { slug, country_code, description, name, document_year, category } = listItem;
```

Comment:
@PatFawbertMills - I've taken country_name out of here - should I 
1. Update the components that use this to remove the country_name as well - or - 
2. Move the responsibility of finding the name into the parent compoents?

Reply:
More inclined to go with 1 - this is more elegant in my opinion. I don't tend to like overloading parent components with concerns of their children.

Reply:
done